core/vibe/core/net.d

```diff
@@ -432,7 +432,14 @@ interface UDPConnection {
 	/// ditto
 	ubyte[] recv(Duration timeout, ubyte[] buf = null, NetworkAddress* peer_address = null);
 
-	/** Become member of IP multicast group
+	/** Set Differentiated Services Code Point
+
+		Used for Quality of Service. 6bit number, higher is more priority.
+		see https://www.iana.org/assignments/dscp-registry/dscp-registry.xhtml
+	*/
+	void setDSCP(uint codePoint);
```

Comment:
Since this is unlikely to change on the protocol level, doesn't it make sense to use `ubyte` here?